migrate/011_add_ipv4.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table(:sshable) do
+      set_column_allow_null :host
+    end
+    alter_table(:vm_host) do
+      add_column :provider, :text, collate: '"C"'
+    end
+    alter_table(:vm) do
+      add_column :local_vetho_ip, :text, collate: '"C"'
+    end
+    create_table :address do
+      column :id, :uuid, primary_key: true, default: Sequel.lit("gen_random_uuid()")
+      column :provider, :text, collate: '"C"', null: false
+      column :cidr, :cidr, null: false, unique: true
+      foreign_key :src_host_id, :vm_host, type: :uuid, null: false
+      foreign_key :dst_host_id, :vm_host, type: :uuid, null: false
+    end
+    create_table :assigned_address do
+      column :id, :uuid, primary_key: true, default: Sequel.lit("gen_random_uuid()")
+      column :cidr, :cidr, null: false, unique: true
+      foreign_key :address_id, :address, type: :uuid, null: false
+      foreign_key :dst_vm_id, :vm, type: :uuid, null: false
+    end
+    create_table :host_address do
+      column :id, :uuid, primary_key: true, default: Sequel.lit("gen_random_uuid()")
+      column :cidr, :cidr, null: false, unique: true
```

Comment:
Is it same with `.address.cidr`

Reply:
Looking into how Mars gave us the servers with 1 subnet that has the server ip in it, I now think that, no they are not. Address will have /~32 subnet while host_address will be /32